crowbar_framework/app/helpers/tag_helper.rb

```diff
@@ -45,9 +45,9 @@ def hash_to_ul(hash)
     )
   end
 
-  # Directly generate a tag for the glyphicons web font icons
+  # Directly generate a tag for the fontawesome web font icons
   def icon_tag(icon, text = nil, options = {})
-    options[:class] = [options[:class], "glyphicon", "glyphicon-#{icon}"].compact.join(" ")
+    options[:class] = [options[:class], "fa", "fa-#{icon.to_s.gsub("_", "-")}"].compact.join(" ")
```

Comment:
Performance/StringReplacement: Use `tr` instead of `gsub`. (https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)